rpcs3/rpcs3_app.cpp

```diff
@@ -282,7 +282,7 @@ void rpcs3_app::InitializeCallbacks()
 			switch (type)
 			{
 			case 0:
-				((gs_frame*)gameWindow)->progress_reset();
+				((gs_frame*)gameWindow)->progress_reset(value);
```

Comment:
does this not throw a warning about conversion into to bool?

Reply:
not in VS